support-files/mysql.server.sh

```diff
@@ -377,7 +377,7 @@ case "$mode" in
       fi
     else
       # Try to find appropriate mysqld process
-      mysqld_pid=`pgrep $libexecdir/mysqld`
+      mysqld_pid=`pgrep -f $libexecdir/mysqld`
```

Comment:
Can just `pgrep mysqld` be used here ?

Reply:
`pgrep` was already used but it was causing a problem that's why I submitted this PR

Reply:
I have not read the full script but I would leave the full path, this way we are not touching any eventual other mysqld process that was not started by this script.

Reply:
I agree with @fauust keeping the full path is safer then removing it.